portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.types.common.models.videos;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Objects;
+
+public class VideoModel {
```

Comment:
For the video content, can we use the schema.org model https://schema.org/VideoObject ?

Reply:
Hm, I am not sure what the way to go is here. When I implemented the model I had a look the schema.org model and decided to keep the implementation consistent with the already existing photo model. Nevertheless it might make sense for the future to go with the schema.org version.

Reply:
Yeah, the original photo model was based on a POJO approach. However, playlists and other more recent models have been able to incorporate some Schema.org models.  Once we incorporate video playlists, we're going to end up having to merge these two. 

Could we use the schema.org model just for the video object for this initial CL  since it's only a few fields we'd want to map for the moment?

Reply:
Adjusted the Model to be based on the schema.org model. We should probably think about the folder structure for the models, especially in regards of dependencies (e.g. in which folder should a MediaObject go?).

Reply:
LGTM. Free free to open an issue for the data model organization.